src/components/CreateForm/CreateForm.scss

```diff
@@ -0,0 +1,20 @@
+@import './src/styles/variebles';
+
+
+#form-dialog-title, button, input,
+.MuiListItem-root.Mui-selected {
+  color: $white !important;
```

Comment:
https://material-ui.com/styles/api/#makestyles-styles-options-hook

we can override styles with material `makeStyles` hook and can avoid using `! important`